milli/src/update/word_prefix_docids.rs

```diff
@@ -124,7 +125,9 @@ fn write_prefixes_in_sorter(
 ) -> Result<()> {
     for (key, data_slices) in prefixes.drain() {
         for data in data_slices {
-            sorter.insert(&key, data)?;
+            if valid_lmdb_key(&key) {
+                sorter.insert(&key, data)?;
+            }
```

Comment:
@MarinPostma, I understand that the fix is here. The unwraps later are considered unreachable.

Reply:
I removed the unwraps and replaced them back by the `?` operator. That was a mistake.